python-nlp-service/nlp_processor.py

```
@@ -1,116 +1,90 @@
-    # 쉼표 포함된 숫자 → "31,231" 등
-    if ',' in token and token.replace(',', '').isdigit():
-        return True
-    # 순수 숫자 (정수)만 포함된 토큰 → "2024", "518"
-    if token.isdigit():
-        return True
-    return False
-
-# --- 모델 및 데이터 로드 함수 (FastAPI startup 이벤트에서 호출) ---
-def load_models_and_data():
-    global okt, tfidf_vectorizer, stopwords_set
+from scripts.tokenizer_utils import dummy_tokenizer
```

Ruff (0.12.2)

`scripts.tokenizer_utils.dummy_tokenizer` imported but unused

Remove unused import: `scripts.tokenizer_utils.dummy_tokenizer`

(F401)